src/operations/blas3/gemm_local.hpp

```diff
@@ -326,6 +326,32 @@ class Gemm<input_t, output_t, DoubleBuffer, NbcA, NbcB, ClSize, TileType,
   }
 
  private:
+  /* @brief If beta is not zero then this function will load in values from C,
```

Comment:
If you want this to be Doxygen you need to start with `/**`

Reply:
Fixed!